clvm/costs.py

```diff
@@ -5,11 +5,11 @@
 LISTP_COST = 1
 
 ARITH_BASE_COST = 1
-ARITH_COST_PER_LIMB_DIVIDER = 32
+ARITH_COST_PER_LIMB_DIVIDER = 16
```

Comment:
All these dividers mean how many bytes of atom you need to pay 1 in cost. So, lower divider means you pay more for memory. Some of these were set a bit high even for powerful computers, but especially for RPi where memory is expensive.

I think it still makes sense since most normal-sized integers will still be free from a cost-per-byte perspective.

Reply:
Thanks - I recognize the limb nomenclature from GMP - do you think the use is complicated enough to warrant a comment?

Reply:
something I've been wanting to do is to rename `LIMB` to `BYTE`, because we actually charge for leading zero bytes now, which technically don't turn into limbs

Reply:
I'm fine with the `LIMB` to `BYTE` change. Feel free to include it in this change if you want.